db-service/test/cqn4sql/model/keyless.cds

```diff
@@ -0,0 +1,17 @@
+// path expressions along `Authors:author` are not possible
+entity Books {
+  key ID : Integer;
+  title  : String;
+  stock  : Integer;
+  author : Association to Authors;
+  authorWithExplicitForeignKey: Association to Authors { ID };
```

Comment:
If there was a calculated element
```
  authorName : author.name;
```
Would selecting `authorName` in a query have to be handled separately?

Reply:
good point, I added this as a test case → does not need to be handled separately :)